styles.css

```diff
@@ -91,3 +87,220 @@ ul {
   align-items: center;
   margin: auto;
 }
+
+/* css for the second part of the project */
+
+.works {
+  display: grid;
+  text-align: center;
+  justify-content: center;
+}
+
+.recent-works {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+  background-image: url("line-works.png");
+  background-size: 100px;
+  background-repeat: no-repeat;
+  background-position: bottom;
+  padding-bottom: 50px;
+  margin-left: 50px;
+  margin-right: 50px;
+}
+
+.img-wrapper {
+  height: 252px;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+  background-color: #ebf0ee;
+}
+
+.card {
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  flex-direction: column;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+  height: 474px;
+  width: 327px;
+}
+
+ul {
+  padding-left: 0;
+}
+
+.card-name {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 24px;
+  line-height: 32px;
+  color: #3a4a42;
+  align-items: center;
+  justify-content: center;
+}
+
+.technologies {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  list-style-type: none;
+  gap: 15px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 16px;
+  color: #3a4a42;
+}
+
+.tech-tag {
+  background: #ebf0ee;
+  border-radius: 4px;
+  padding: 8px 12px;
+}
+
+.see-project {
+  padding: 12px;
+  gap: 10px;
+  width: 122px;
+  height: 48px;
+  background: #36b37f;
+  border-radius: 4px;
+  color: #fff;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  outline: 0;
+  border: 0;
+}
+
+.see-project:hover {
+  box-shadow: 8px 16px 10px rgba(54, 179, 127, 0.24);
+}
```

Comment:
- [ ] [OPTIONAL] I'd like to apologize as the previous reviewers forgot to mention this but please try to stick with the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1-468&t=Bjxe1oWlti3Nl2On-0)🎨 as much as possible.
    - This is what the design specifies for the different states of the button:
      ![firefox_kz6PEcoelc](https://user-images.githubusercontent.com/104968236/227737823-4354928e-8f3b-4a2c-9d17-af36ecfde213.png)
    - You can see there's no box shadow so please fix that. 👍
        - Just change the background color to `#36B37F` when hovered and to `#008552` when clicked